lib/fast_stats/mean.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# A class for managing the arithmetic and geometric means
+# of positive numbers
```

Comment:
If you do a set line-wrap as @reillysiemens mentioned earlier, make sure it's the same wrapping here

Reply:
What is a "a set line-wrap"? Not sure what hits is referencing.